web-share/share-url-empty-manual.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Test: Share with empty URL</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
```

Comment:
Oh, I see... Ok, but I shouldn't have to go look at the `setupManualShareTest()` call site to understand the test. Consider, there is a boolean trap as the first argument of `setupManualShareTest`, no idea what the second and third arguments are, but can assume what `getAbsoluteUrl()` does: 

```
 setupManualShareTest(false, '', '', getAbsoluteUrl(url))
```

Thus, for ones like the above, or globally, I would expect a comment explaining what is being tested (if it's not immediately obvious from just looking at `setupManualShareTest()`).

If `setupManualShareTest()` is something that we control, then we should fix it to use a dictionary so it's actually clear what each argument slot does. 

Make sense?